primitives/runtime/src/generic/header.rs

```diff
@@ -22,8 +22,8 @@ use crate::{
 	generic::Digest,
 	scale_info::TypeInfo,
 	traits::{
-		self, AtLeast32BitUnsigned, Hash as HashT, MaybeDisplay, MaybeSerialize,
-		MaybeSerializeDeserialize, Member, SimpleBitOps,
+		self, AtLeast32BitUnsigned, Hash as HashT, HashOutput, MaybeDisplay, MaybeFromStr,
```

Comment:
```suggestion
		self, AtLeast32BitUnsigned, Hash as HashT, MaybeDisplay, MaybeFromStr,
```
If you apply the suggestions this is not required